config/server.js

```diff
@@ -1,13 +1,12 @@
 // config used by server side only
-
+const dbConectString = process.env.DB_STRING || null;
```

Comment:
setting DB_STRING environmental variable will override any other DB settings variable.

this helped me greatly getting a new project working with the hosted mongoDB atlas service